apx_gui/core/apx_entities.py

```diff
@@ -179,16 +183,24 @@ def __init__(
         self.enter_command: list[str] = enter_command
         self.exported_programs: dict[str, dict[str, str]] = exported_programs or {}
 
-    def create(self) -> tuple[bool, "Subsystem"]:
-        new_command: str = (
-            f"subsystems new --name '{self.name}' --stack '{self.stack.name}'"
+    def create(
+        self,
+        _terminal,
+    ) -> tuple[bool, "Subsystem"]:
+        new_command = (
+            f"{self._get_apx_command()}",
+            "subsystems",
+            "new",
+            "--name",
+            f"{self.name}",
+            "--stack",
+            f"{self.stack.name}",
         )
         # the following apx command is safe to ignore errors, we´ll check the
         # subsystem status by getting the list of subsystems
-        new_res: tuple[bool, str] = self._run_apx_command(new_command, True)
-        if not new_res[0]:
-            return new_res[0], self
+        self.run_vte_command(new_command, _terminal, self._Create_Callback)
 
+    def _Create_Callback(self,*args):
```

Comment:
I would keep this lowercased

Reply:
Alrighty, I normally use the VariableName style but  I'll do that!